pinot-segment-local/src/main/java/org/apache/pinot/segment/local/recordtransformer/RecordEnricher.java

```diff
@@ -16,25 +16,21 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.pinot.spi.recordenricher;
-
-import java.util.List;
-import org.apache.pinot.spi.data.readers.GenericRow;
-
+package org.apache.pinot.segment.local.recordtransformer;
 
 /**
  * Interface for enriching records.
  * If a column with the same name as the input column already exists in the record, it will be overwritten.
  */
-public interface RecordEnricher {
-  /**
-   * Returns the list of input columns required for enriching the record.
-   * This is used to make sure the required input fields are extracted.
-   */
-  List<String> getInputColumns();
-
-  /**
-   * Enriches the given record, by adding new columns to the same record.
-   */
-  void enrich(GenericRow record);
-}
+//public interface RecordEnricher {
```

Comment:
Please cleanup the commented lines, and files that are no longer needed. Same for other places